builder.js

```diff
@@ -118,7 +118,10 @@ Builder.prototype.build = function build(spec, callback) {
         // already been downloaded
         //
         cleanup();
-        writeStream.end({ eventType: 'complete' }, callback);
+        writeStream.end({
+          eventType: 'complete',
+          message: 'Assets build completed'
```

Comment:
nit: "Assets" here is a little misleading no? How about `message: 'carpenterd build completed'`?

Reply:
@indexzero i suppose. I was trying to disambiguate from `webpack build completed` and since this is a thing that publishes assets /shrug. Im ok with`carpenterd-worker build completed` i guess.

Reply:
I kinda like `carpenterd-worker build completed` since this message is really "I'm fully done with this specific build"